src/extensions/sticky-header/bootstrap-table-sticky-header.css

```diff
@@ -17,6 +17,6 @@
   background: #e9ecef;
 }
 
-.fix-sticky table thead.thead-light {
+.fix-sticky table thead.thead-dark {
```

Comment:
Why is this change? The hex color is the same

Reply:
The last version was wrong: https://github.com/wenzhixin/bootstrap-table/blob/develop/src/extensions/sticky-header/bootstrap-table-sticky-header.css#L16